server/src/main/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationFileCreator.java

```diff
@@ -23,12 +23,13 @@
 public class CloudFormationFileCreator {
     public static final String CLI_COMMAND_CREATESTACK = "aws cloudformation deploy ";
     public static final String CLI_COMMAND_DELETESTACK = "aws cloudformation delete-stack ";
-    public static final String CLI_COMMAND_DELETEBUCKET = "aws s3api delete-bucket --bucket ";
+    public static final String CLI_COMMAND_DELETEBUCKET = "aws s3 rb s3://";
     public static final String CLI_PARAM_STACKNAME = "--stack-name ";
     public static final String CLI_PARAM_TEMPLATEFILE = "--template-file ";
-    public static final String CLI_PARAM_PARAMOVERRIDES = "--parameter-overrides ";
+    public static final String CLI_PARAM_PARAMOVERRIDES = "--parameter-overrides";
     public static final String CLI_PARAM_CAPABILITIES = "--capabilities";
     public static final String CLI_PARAM_BUCKET = "--bucket ";
+    public static final String CLI_PARAM_FORCE = "--force";
```

Comment:
You could consider add a blank in the end of these strings, so you could save additional blanks in your string buildings.

Reply:
I'm gonna do this in a future PR, all these Strings should be standardised anyways since some of them have a blank and some of them don't.